include/zephyr/net/tftp.h

```diff
@@ -20,31 +20,22 @@ extern "C" {
 #endif
```

Comment:
Could you add the `net: ...` prefix to the commit subject

Reply:
Will be `net: tftp_client: Add client context`